examples/Retrieval-Models.ipynb

```diff
@@ -0,0 +1,1528 @@
+{
```

Comment:
<p><code>It is Thus, these systems are often composed of two stages</code> reads a little weird.  Maybe something like <code>Because of this, these systems are often composed of two stages</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
thanks.